docs/UsersGuide/source/Introduction.rst

```diff
@@ -1,177 +1,330 @@
-and CCPP technical aspects are described in the `CCPP Technical Documentation
-<https://ccpp-techdoc.readthedocs.io/en/v5.0.0/>`_. The model namelist has many settings
-beyond the physics options that can optimize various aspects of the model for use with each
-of the supported suites. 
-
-The SRW App supports the use of both GRIB2 and :term:`NEMSIO` input data. The UFS Weather Model
-ingests initial and lateral boundary condition files produced by :term:`chgres_cube` and outputs files in
-netCDF format on a specific projection (e.g., Lambert Conformal) in the horizontal and model
-levels in the vertical.
-
-Post-processor
-==============
+(:term:`FV3`) dynamical core configured with a Limited Area Model (LAM) capability (:cite:t:`BlackEtAl2020`).
+The dynamical core is the computational part of a model that solves the equations of fluid motion. A User’s Guide for the UFS :term:`Weather Model` can be found `here <https://ufs-weather-model.readthedocs.io/en/ufs-v2.0.0/>`__. 
+
+Common Community Physics Package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `Common Community Physics Package <https://dtcenter.org/community-code/common-community-physics-package-ccpp>`_ (:term:`CCPP`) supports interoperable atmospheric physics and Noah Multi-parameterization (Noah MP) Land Surface Model options. Atmospheric physics are a set of numerical methods describing small-scale processes such as clouds, turbulence, radiation, and their interactions. The SRW release includes an experimental physics version and an updated operational version. 
+
+Data Format
+^^^^^^^^^^^^^^^^^^^^^^
+
+The SRW App supports the use of both :term:`GRIB2` and :term:`NEMSIO` input data. The UFS Weather Model ingests initial and lateral boundary condition files produced by :term:`chgres_cube`. 
+
+
+Unified Post-Processor (UPP)
+--------------------------------
 
-The SRW Application is distributed with the Unified Post Processor (:term:`UPP`) included in the
-workflow as a way to convert the netCDF output on the native model grid to GRIB2 format on
-standard isobaric vertical coordinates. UPP can also be used to compute a variety of useful
-diagnostic fields, as described in the `UPP user’s guide <https://upp.readthedocs.io/en/upp-v9.0.0/>`_.
+The `Unified Post Processor <https://dtcenter.org/community-code/unified-post-processor-upp>`__ (:term:`UPP`) processes raw output from a variety of numerical weather prediction (:term:`NWP`) models. In the SRW, it converts data output formats from netCDF format on the native model grid to GRIB2 format. The UPP can also be used to compute a variety of useful diagnostic fields, as described in the `UPP User’s Guide <https://upp.readthedocs.io/en/upp-v9.0.0/>`_. Output from the UPP can be used with visualization, plotting, and verification packages, or for further downstream post-processing (e.g., statistical post-processing techniques).
```

Comment:
The use of "native model grid" here could be a bit misleading.   The SRW App has what's called the write component which takes the native FV3 gnomonic grid and converts it to a number of regional projections (e.g., lat/lon, rotated lat/lon, Lambert conformal).  So UPP is reading in data on this interpolated grid, which isn't truly the native model grid in this case.